flink_jobs_v2/batch_status/src/main/java/argo/batch/StatusMetric.java

```diff
@@ -226,4 +226,74 @@ public String toString() {
         return "(" + this.group + "," + this.service + "," + this.hostname + "," + this.metric + "," + this.status + "," + this.timestamp + ","
                 + this.dateInt + "," + this.timeInt + "," + this.prevState + "," + this.prevTs + "," + this.actualData + "," + this.ogStatus + "," + this.ruleApplied + "," + this.info + "," + this.tags + ")";
     }
+    @Override
```

Comment:
Same here, do we need this ?

Reply:
We do not need this right now as no StatusMetric data are compared. It will be in need in the next steps as List.containsAll() calls an equals() and fails as it compares objects not by property but by address. We need to compare the objects based on their properties so we replace them. In this PR it will be removed as it is uneccessary